src/demo/renderer.js

```diff
@@ -317,28 +317,69 @@ const drawFrame = state => {
 const drawPondFishImages = () => {
   const canvas = getState().canvas;
   const ctx = canvas.getContext('2d');
+
+  const fishBounds = [];
+
   getState().pondFish.forEach(fish => {
+    const pondClickedFish = getState().pondClickedFish;
+    const pondClickedFishUs = pondClickedFish && fish.id === pondClickedFish.id;
+
+    const swayMultipleX = pondClickedFishUs ? 120 : 120;
```

Comment:
are both values 120 on purpose?

Reply:
Great question.  For now, yes.  I experimented with reducing the sway when a fish is selected, but this causes the newly-enlarged fish to jump to a different location to where they just were, because it puts them on an entirely new sway path.  I wanted to leave the code like this until I thought of a better approach, but I'll just remove it for now.